haystack/nodes/file_converter/parsr.py

```diff
@@ -71,11 +71,15 @@ def __init__(
         :param add_page_number: Adds the number of the page a table occurs in to the Document's meta field
                                 `"page"`.
         :param extract_headlines: Whether to extract headings from the PDF file.
+        :param timeout: How many seconds to wait for the server to send data before giving up,
+            as a float, or a :ref:`(connect timeout, read timeout) <timeouts>` tuple.
+            Defaults to 10 seconds.
+        :type timeout: Union[float, Tuple[float, float]]
```

Comment:
No need to add the type here as we already have it in the type annotation in the methods signature